### PR TITLE
Constructor

### PR DESCRIPTION
**Pull request by baronin**
Еще в работе

### CODE REVIEW

File: Constructor/index.js [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+if (submit) {
+  submit.addEventListener('click', function () {
+    if (inputYesText.checked) {
+      if (textArea.value) {
+        getData();
+        body.style.backgroundColor = bgColor.value;
+        body.style.color = textColor.value;
+        return;
+      } else {
+        console.log(' else');
```
Comment (codeismagicschool) [inline Constructor/index.js]: Please remove all ```console.log```

File: Constructor/index.js [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+      },
+    });
+    setTimeout(function () {
+      mySwiper.update();
+    }, 1000);
+  }
+}
+
+if (submit) {
+  submit.addEventListener('click', function () {
```
Comment (codeismagicschool) [inline Constructor/index.js:207]: Please save all event handlers to variables: 
```javascript
const Events = {
 CLICK: 'click',
 ...
 };
 const onSubmit = (event) => {...}
submit?.addEventListener?.(Events.CLICK, onSubmit)
```

File: Constructor/style.css [hunk 3]
```diff
@@ -0,0 +1,296 @@
...
+  background-size: contain;
+  cursor: pointer;
+  height: 17px;
+}
+input[type='radio']:checked + label {
+  background: url('img/radio_on.png') transparent no-repeat;
+  background-size: contain;
+}
+input[type='radio']:checked + label + img {
+  display: block !important;
```
Comment (codeismagicschool) [inline Constructor/style.css:28]: Please don`t use ```!important```

File: Constructor/style.css [hunk 4]
```diff
@@ -0,0 +1,296 @@
...
+}
+input[type='radio'] {
+  display: none;
+}
+#menuNav img {
```
Comment (codeismagicschool) [inline Constructor/style.css:33]: it's better to use className instead of id

File: Constructor/style.css [hunk 5]
```diff
@@ -0,0 +1,296 @@
...
+  color: inherit;
+}
+
+label {
+  color: white;
+}
+
+.title {
+  border-radius: 5px;
+  border: solid #d6d6d6 1px;
```
Comment (codeismagicschool) [inline Constructor/style.css:80]: NIT: Try to use variables for colors: https://developer.mozilla.org/ru/docs/Web/CSS/Using_CSS_custom_properties

**Review by codeismagicschool — COMMENTED** [review]
Great job

File: Constructor/index.js (hunk 1 above)
Comment (baronin) [reply Constructor/index.js]: Sorry ( I removed all logs

File: Constructor/style.css (hunk 3 above)
Comment (baronin) [reply Constructor/style.css:28]: these are not my styles, but sorry you see it ^(

File: Constructor/style.css (hunk 4 above)
Comment (baronin) [reply Constructor/style.css:33]: I removed it

File: Constructor/style.css (hunk 5 above)
Comment (baronin) [reply Constructor/style.css:80]: Yep! Cool

**Pull request merged**